lib/livebook/application.ex

```diff
@@ -48,7 +48,7 @@ defmodule Livebook.Application do
           # Start the Endpoint (http/https)
           # We skip the access url as we do our own logging below
           {LivebookWeb.Endpoint, log_access_url: false}
-        ] ++ app_specs()
+        ] ++ app_specs() ++ identity_provider()
```

Comment:
We must start the provider before the endpoint (because the endpoint may receive requests that require the provider).